homeassistant/components/flic/binary_sensor.py

```diff
@@ -129,6 +129,7 @@ class FlicButton(BinarySensorEntity):
     def __init__(self, hass, client, address, timeout, ignored_click_types):
         """Initialize the flic button."""
 
+        self._attr_unique_id = f"flic_{address.lower().replace(':', '')}"
```

Comment:
We don't need to add the integration domain to the unique_id. The entity registry is aware of both the integration domain and the platform domain of the entity.

Reply:
I think you should be using `homeassistant.helpers.device_registry.format_mac` to create the unique_id form the bluetooth MAC address.

https://developers.home-assistant.io/docs/entity_registry_index#unique-id-requirements

Reply:
Thanks for the pointers, done.